app/lib/actions/katello/host/update.rb

```diff
@@ -30,6 +30,10 @@ def plan(host, consumer_params = nil)
           plan_self(:hostname => host.name)
         end
 
+        def resource_locks
+          :update
```

Comment:
No, see the existing examples in the code base. I'm guessing Object.available_locks does need to return an array.